tests/serverpod_test_server/test/generated_methods/to_json_test.dart

```diff
@@ -611,8 +611,7 @@ void main() {
     });
   });
 
-  test(
-      'Given an object with a server only field then that field is not serialized.',
+  test('Given an object with a server only field then field is serialized.',
```

Comment:
Suggestion: This is actually a nested field, please reflect that in the description.

It would be good to add a field for the first level to ensure we serialize them.